src/DynamoPackages/PackageLoader.cs

```diff
@@ -386,16 +389,28 @@ public void LoadPackages(IEnumerable<Package> packages)
         }
 
         /// <summary>
-        /// Scans and compiles any new packages contained in new package paths that are added.
-        /// Note: This method will still reload ALL packages in the VM even those that are not new.
+        /// This method is called when custom nodes and packages need to be reloaded if there are new package paths.
         /// </summary>
         /// <param name="loadPackageParams"></param>
-        private void LoadNewPackages(LoadPackageParams loadPackageParams)
```

Comment:
Why has this been removed? This method was added for a reason as part of a recent fix. It was specifically done as part of https://github.com/DynamoDS/Dynamo/pull/11835/files for a bug fix to prevent reloading of packages ⚠️